tests/taskadmins/docker-compose.yml

```diff
@@ -2,19 +2,12 @@ version: "3"
 services:
   mesos-term:
     environment:
-      - MESOS_TERM_COMMAND=/bin/bash
       - MESOS_TERM_ENABLE_PER_APP_ADMINS=true
       - MESOS_TERM_ENABLE_RIGHTS_DELEGATION=true
       - MESOS_TERM_LDAP_URL=ldap://openldap
       - MESOS_TERM_LDAP_BASE_DN=dc=example,dc=com
       - MESOS_TERM_LDAP_USER=cn=admin,dc=example,dc=com
       - MESOS_TERM_LDAP_PASSWORD=password
-      - MESOS_TERM_JWT_SECRET=abcd
-      - MESOS_TERM_MESOS_MASTER_URL=http://mesos-master:5050
-      - MESOS_TERM_MESOS_STATE_CACHE_TIME=60
-      - MESOS_TERM_MESOS_AGENT_PRINCIPAL=admin
-      - MESOS_TERM_MESOS_AGENT_PASSWORD=password
-      - MESOS_TERM_SESSION_SECRET=abcd
       - MESOS_TERM_SUPER_ADMINS=admins
       - MESOS_TERM_ALLOWED_TASK_ADMINS=devops
       - MESOS_TERM_AUTHORIZE_ALL_SANDBOXES=true
```

Comment:
Why is it not needed to specify SUITE here? Maybe if it's because of defaults, maybe it can be put there anyway for clarity sake

Reply:
the SUITE env variable is actually not necessary :) I'll remove it or put it everywhere